linux/LinuxProcessList.c

```diff
@@ -1640,8 +1684,13 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_
       }
       #endif
 
-      if (ss->flags & PROCESS_FLAG_LINUX_CGROUP) {
-         LinuxProcessList_readCGroupFile(lp, procFd);
+      /* The process' cgroup name is required for checking whether it runs in a container */
+      LinuxProcessList_readCGroupFile(lp, procFd);
+      char *cgroup = lp->cgroup;
+      if ( lp->cgroup == NULL
+         || (!(String_startsWith(cgroup, "/user") || String_startsWith(cgroup, "/machine") || String_startsWith(cgroup, "/system"))) ){
```

Comment:
```suggestion
      if (!lp->cgroup || 
         !(String_startsWith(cgroup, "/user") || String_startsWith(cgroup, "/machine") || String_startsWith(cgroup, "/system")) ) {
```

Logic operators should appear at EOL.
Skip unnecessary (((parens))) …
There's a blank before the opening brace of a blcok (recheck the style guide and the existing source).

Reply:
> Logic operators should appear at EOL.

Alright, got it. I think I got confused because of the convention used [here](https://github.com/htop-dev/htop/blob/main/ProcessList.c#L400)